packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -829,6 +829,14 @@
             }
         }
     },
+    "product_tags_title": "Product Tags",
```

Comment:
Could you add a description for each of your translations?
It adds some context for translators.